src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+import '../styles/account_menu_dropdown.css'
+
+class AccountMenuDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggleMenu = this.toggleMenu.bind(this);
+    this.state = {
+      menuActive: false
+    };
+  }
+
+  toggleMenu() {
+    let menuState = !this.state.menuActive;
+    this.setState({
+      menuActive: menuState
+    });
+  }
+
+  render() {
+    let menu;
```

Comment:
I think this ```menu``` has to be defined by ```let``` instead of ```const``` because we are using an if/else statement...